support/releaseinfo/registry_mirror_provider.go

```diff
@@ -27,18 +27,29 @@ func (p *RegistryMirrorProviderDecorator) Lookup(ctx context.Context, image stri
 	p.lock.Lock()
 	defer p.lock.Unlock()
 
+	// Replace the image's registry source with the first replacement prior to looking up the image
+	for registrySource, registryDest := range p.RegistryOverrides {
+		for _, registryReplacement := range registryDest {
+			image = strings.Replace(image, registrySource, registryReplacement, 1)
```

Comment:
I don't think the purpose of the multiple replacement entries is to do multiple replacements on the original image string, rather to provide alternatives if one of them fails.

I think a better implementation is to iterate through all the replacements, replace the string in the image, then do a `p.Delegate.Lookup` on the replaced image ref. If that returns an error, then try the next replacement. If they all fail, return error.